pkg/testing/integration/program.go

```diff
@@ -302,6 +302,30 @@ func (opts ProgramTestOptions) With(overrides ProgramTestOptions) ProgramTestOpt
 	if overrides.RunBuild {
 		opts.RunBuild = overrides.RunBuild
 	}
+	if overrides.ExpectFailure {
+		opts.ExpectFailure = overrides.ExpectFailure
```

Comment:
Nit: all of these bools could just be `opts.Foo = true`.

Reply:
Fair point - but just for consistency I'll leave as is.